CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
-## [unreleased]
```

Comment:
I'm not sure what its for. I don't see it show up in the rendered notes. Maybe @patflynn knows?